ucfcs/GrowthPlus
# Julio/quiz flashcard tables

This PR implements quiz schema, flashcard schema updates lesson schema and roadmap schema. 
 
To test:
Uninstall the app. 
Build and run the app, make sure it does not crash. 
Find the realm file and inspect that the default data for the mentioned schemas are there. 

Note: 
The schema service files still need implemented, will open a ticket for it.
